dbt/adapters/bigquery/impl.py

```diff
@@ -248,6 +248,17 @@ def get_columns_in_relation(self, relation: BigQueryRelation) -> List[BigQueryCo
             logger.debug("get_columns_in_relation error: {}".format(e))
             return []
 
+    @available
+    def alias_column(self, column) -> Dict[str, str]:
```

Comment:
Where would we be checking to see if a column name needs aliasing, on account of starting with `__`? Are you thinking that we'd add calls to `alias_column` within the code that passes the `column_name` value into generic test macro definitions?

Reply:
Hi Jeremy, thanks for the response and questions! Your example of how the test would be defined in the yml is correct for what I was thinking for the user experience. 

As for checking if the column name needs aliasing, yes, the idea was that generic tests could call this method and remove the leading underscore. It may need to be done differently depending on the macro which is why I have it returning a dictionary of the original column name, the aliased column name, or the actual aliasing that would need to be included in a select statement: `select _FILE_NAME as FILE_NAME`. Here is an example of how I updated `dbt_utils.unique_combination_of_columns` to use the method: 

```
{% macro bigquery__test_unique_combination_of_columns(model, combination_of_columns, quote_columns=false) %}

{%- set select_list=[] %}
{% if not quote_columns %}
        {% set group_list = combination_of_columns %}
        {% for column in combination_of_columns -%}
            {% set column = adapter.alias_column( column )['statement'] or column %}
            {% set select_list = select_list.append( column ) %}
        {%- endfor %}
{% elif quote_columns %}
        {% for column in combination_of_columns -%}
            {% set select_list = select_list.append( adapter.quote(column) ) %}
            {% set group_list = select_list %}
        {%- endfor %}
{% else %}
    {{ exceptions.raise_compiler_error(
        "`quote_columns` argument for unique_combination_of_columns test must be one of [True, False] Got: '" ~ quote ~"'.'"
    ) }}
{% endif %}

{%- set group_csv, select_csv=group_list | join(', '), select_list | join(', ') %}


with validation_errors as (

    select
        {{ select_csv }}
    from {{ model }}
    group by {{ group_csv }}
    having count(*) > 1

)

select *
from validation_errors

{% endmacro %}
```